files/build_templates/sonic_debian_extension.j2

```diff
@@ -462,11 +462,10 @@ sudo LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS ta
 {% if install_kubernetes == "y" %}
 ## Pull in kubernetes docker images
 echo "pulling universal k8s images ..."
+K8S_ARCH=$([ "${CONFIGURED_ARCH}" == "armhf" ] && echo "arm64" || echo "${CONFIGURED_ARCH}")
```

Comment:
>K8S_ARCH [](start = 0, length = 8)

Is it FLANNEL_ARCH ?

Reply:
Guess, yes for now.  Do you want this changed ? I could.

Reply:
Yes, please fix it.

---
In reply to: [439624555](https://github.com/Azure/sonic-buildimage/pull/4768#discussion_r439624555) [](ancestors = 439624555)

Reply:
done